roofit/batchcompute/src/RooBatchCompute.cu

```diff
@@ -45,6 +44,9 @@ inline static void __checkCudaErrors(cudaError_t error, std::string func, std::s
 namespace RooBatchCompute {
 namespace RF_ARCH {
 
```

Comment:
Do you want maybe to configure these two values outside as a global configuration flags?

Reply:
At a later point this will be done. After the rush to CHEP, the CUDA library will get revisited again anyway, especially with the openlab student.